packages/terra-site/src/examples/list/ListIndexRoutes.jsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { Route } from 'react-router';
+
+// Form Component Indexes
+import List from './components/List';
+import MultiSelect from './components/MultiSelect';
+import MultiSelectMaxSelection from './components/MultiSelectMaxSelection';
+import SingleSelect from './components/SingleSelect';
+
+const routes = (
+  <div>
+    <Route path="/site/list/list" component={List} />
+    <Route path="/site/list/multi-select" component={MultiSelect} />
```

Comment:
My vote would be to combine the MultiSelect examples on one example page.

Reply:
https://github.com/cerner/terra-core/pull/853/commits/9c3f80105944f72a9da5a84cb7b941399f8f31a8